airflow/providers/google/cloud/triggers/dataproc.py

```diff
@@ -84,3 +84,64 @@ async def run(self):
                     raise AirflowException(f"Dataproc job execution failed {self.job_id}")
             await asyncio.sleep(self.polling_interval_seconds)
         yield TriggerEvent({"job_id": self.job_id, "job_state": state})
+
+
+class DataprocClusterTrigger(BaseTrigger):
```

Comment:
does it make sense to extend this class from DataprocBaseTrigger instead of BaseTrigger https://github.com/apache/airflow/blob/main/airflow/providers/google/cloud/triggers/dataproc.py#L31 WDYT?

Reply:
DataprocBaseTrigger is dedicated to e.g. DataprocSubmitJobOperator.
I changed the name to DataprocSubmitTrigger for clarity.